db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ScanCostFunction.java

```diff
@@ -529,7 +533,8 @@ public void generateCost() throws StandardException {
             scanCost.setIndexLookupRows(-1.0d);
             scanCost.setIndexLookupCost(-1.0d);
         } else {
-            lookupCost = totalRowCount*filterBaseTableSelectivity*(openLatency+closeLatency);
+            double lookupCostPerRow = isOlap ? (openLatency+closeLatency) : ((openLatency+closeLatency) / 10);
```

Comment:
Another magic number. Can we either turn this into a tunable parameter (latencyOLTPFactor?) or remove the explicit 10 so that people understand what it's about

Reply:
Ideally, `openLatency` and `closeLatency` can be sampled for OLTP and OLAP periodically when running queries. Then this artificial factor can be removed. For now, I changed it to a constant from a literal. Making it tunable is possible. I record this parameter in DB-11481, too.